.github/workflows/main.yml

```diff
@@ -0,0 +1,23 @@
+name: Main
+
+on: [pull_request, push]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Install dependencies
+        run: yarn
+      - name: Build from src
+        run: |
+          git clone https://github.com/emscripten-core/emsdk.git
```

Comment:
Maybe pin this to release tag or commit or even better use github action for that: https://github.com/mymindstorm/setup-emsdk

Reply:
Great idea!